README.md

```diff
@@ -10,6 +10,12 @@ Linux/Mac: use your favorite package tool. For example, on Ubuntu:
 sudo apt install build-essential pkg-config libusb-1.0-0-dev
 ```
 
+On Linux you can add udev rules in order to use picotool without sudo:
+
+```console
+sudo cp udev/99-picotool.rules /etc/rules/99-picotool.rules
```

Comment:
I thought `/etc/udev/rules.d` was the main place to put rules like this.  I don't see /etc/rules documented [here](https://www.freedesktop.org/software/systemd/man/udev.html).